rclcpp/test/benchmark/CMakeLists.txt

```diff
@@ -26,9 +26,9 @@ if(TARGET benchmark_node)
   target_link_libraries(benchmark_node ${PROJECT_NAME})
 endif()
 
-add_performance_test(benchmark_parameter benchmark_parameter.cpp)
-if(TARGET benchmark_parameter)
-  target_link_libraries(benchmark_parameter ${PROJECT_NAME})
+add_performance_test(benchmark_parameter_client benchmark_parameter_client.cpp)
```

Comment:
Since this is benchmarking the node_parameters interface, how about `benchmark_node_parameters_interface`? Everything else looks good to me

Reply:
4a1d7b6